AllReadyApp/Web-App/AllReady/Startup.cs

```diff
@@ -371,6 +370,23 @@ private IContainer CreateIoCContainer(IServiceCollection services)
                 await sampleData.CreateAdminUser();
             }
         }
+
+        private static void CheckForHangfireDatabaseAndCreateIfItDoesNotExist()
+        {
+            using (var sqlConnection = new SqlConnection("Server=(localdb)\\MSSQLLocalDB;Integrated Security=true;MultipleActiveResultsets=true;"))

```

Comment:
@mgmccarthy it looks like you can give Hangfire a connection string with a database name: http://docs.hangfire.io/en/latest/configuration/using-sql-server.html 

If I'm reading this right, it also creates the Hangfire tables in a schema named Hangfire within the specified database. If that's the case I think we should put the hangfire stuff in the same database as allReady.